src/sdk.rs

```diff
@@ -156,7 +155,6 @@ mod exports {
     }
 }
 
-#[allow(dead_code)]
```

Comment:
Hooray!

It would be great if we could eventually remove all of these that are peppered throughout the codebase.